### DISCUSSION TITLE
How to have an external package working with a M2 Mac?

### DISCUSSION THREAD

**Discussion by trappist1i** [iraf-community/iraf#311] [Q&A]
I installed the Iraf version called iraf-main from https://github.com/orgs/iraf-community/discussions/297
on my new Mac Book Pro with a M2 chip and the OS Ventura 13.4
I then copied an external package of mine into the _extern_ directory.  
When I launch iraf through cl, the external package doesn't appear.
I tried filling extern.pkg with the proper information, but it doesn't work either. 
Any help would be welcome!

**Comment by olebole** [comment]
Did it work before on 2.17 or 2.16.1?

If the package needs compilation, it should be recompiled on your new system to work, i.e. to have ARM binaries. If it still does not work, you would need to debug `extern.pkg` on why the tasks are not loaded. Maybe the paths are not correct there?

I tried it for the "wcstools" package, and it loads well without any modification.
